MySTARS/src/controls/StaffControl.java

```diff
@@ -59,7 +53,8 @@ public static boolean addStudent(Staff currentStaff, String name, String userID,
         Student student;
 		try {
 			student = new Student(name, userID, gender, nationality, schoolID, identificationKey);
-            db.addStudentPassword(userID, userPW);
+            Password.addNewPassword(userID, userPW);
```

Comment:
I don't know this works for the password for the staff. Need to use the old addStudentPassword method.